src/app/pages/send/send-crypto-asset-form/components/recipient-field.tsx

```diff
@@ -22,6 +27,16 @@ export function RecipientField({
   placeholder,
   topInputOverlay,
 }: RecipientFieldProps) {
```

Comment:
I tried a few approaches and decided on this. I'm not sure if it's the best but it's working well. 

This transforms the value directly and updates the `Formik` field.

I'm open to suggestions of a better way here.